### PR TITLE
Logs support

### PR DESCRIPTION
**Pull request by sandeep-paliwal**
Fetch action logs for given aio app

## Description

Added support for fetching logs from CLI.
Command aio app:logs
Added flag limit (-l) to specify the number to latest activations for which to fetch logs(default 1)

## Related Issue

NA

## Motivation and Context

Log Support

## How Has This Been Tested?

Manually

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ x] My code follows the code style of this project.
- [x ] My change requires a change to the documentation.
- [x ] I have updated the documentation accordingly.
- [x ] I have read the **CONTRIBUTING** document.
- [x ] I have added tests to cover my changes.
- [x ] All new and existing tests passed.

### CODE REVIEW

File: src/commands/app/logs.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  async getLogs (limit) {
+    const auth = Buffer.from(this.aioConfig.runtime.auth).toString('base64')
+    const authHandler = {
+      getAuthHeader: ()=>{
+        return Promise.resolve('Basic ' + auth)
+      }
+    }
+    // get activations
+    const options = {apihost: 'runtime.adobe.io',
+    auth_handler: authHandler,
```
Comment (moritzraho) [inline src/commands/app/logs.js]: you can directly pass the auth through `api_key` param

**Comment by purplecabbage** [comment]
Running command in a non-cna I see an error:
```
$ aio app logs
TypeError: Cannot read property 'auth' of undefined
    at Logs.getLogs (/repos/adobe/aio-cli-all/aio-cli-plugin-app/src/commands/app/logs.js:30:53)
    at Logs.run (/repos/adobe/aio-cli-all/aio-cli-plugin-app/src/commands/app/logs.js:25:16)
    at Logs._run (/repos/adobe/aio-cli-all/aio-cli-plugin-app/node_modules/@oclif/command/lib/command.js:44:31)
```
> should this be more descriptive of the problem encountered?

Running when no logs are returned:
```
$ aio app logs
✔ Finished fetching logs!
```
> should this state that no logs were returned? _it seems a little self-congratulatory with the checkmark and no results_ 😆

**Comment by purplecabbage** [comment]
These should maybe be the same thing ... 
```
/repos/adobe/aio-cli-all/gapp [node:v12.3.1] 
$ aio app logs
2019-11-04T19:47:19.319Z       stdout: main called with args: Wilfred - hello
2019-11-04T19:47:19.319Z       stdout: time to say hello to my friend  Wilfred - hello
✔ Finished fetching logs!

/repos/adobe/aio-cli-all/gapp [node:v12.3.1] 
$ aio rt logs -l
activation logs b8b4de7ed2e747e7b4de7ed2e737e7a9
2019-11-04T19:47:19.319Z       stdout: main called with args: Wilfred
2019-11-04T19:47:19.319Z       stdout: time to say hello to my friend  Wilfred
```
I like that your implementation outputs the action name, although it is not obvious what that is.
I like the way the `rt` implementation outputs the activation.

My personal pref would be to see output of:
```
hello:0.0.25:b8b4de...
2019-11-04T19:47:19.319Z       stdout: main called with args: Wilfred
2019-11-04T19:47:19.319Z       stdout: time to say hello to my friend  Wilfred
hello:0.0.25:eb7d99...
2019-11-04T19:47:04.643Z       stdout: main called with args: Lumpy
2019-11-04T19:47:04.643Z       stdout: time to say hello to my friend  Lumpy
```

File: src/commands/app/logs.js (hunk 1 above)
Comment (sandeep-paliwal) [reply src/commands/app/logs.js]: done

File: src/commands/app/logs.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    }
+  }
+
+  async getActivationLogs (activation, ow) {
+    const logOptions = {activationId: activation.activationId}
+    let results = await ow.activations.logs(logOptions)
+    // send fetched logs to console
+    if (results.logs) {
+      results.logs.forEach(function (log) {
+        console.log(log + " - " + activation.name)
```
[the remark this answers is not in the document]
Comment (sandeep-paliwal) [reply src/commands/app/logs.js]: done

**Pull request merged**